docs/manual/external-tools.tex

```diff
@@ -906,44 +906,32 @@
 \end{Verbatim}
 
 \item
-  To use a local version of the Checker Framework that
-  you have built from source (Section~\ref{build-source}):
+  To use a locally-modifed version of the Checker Framework, first run:
 
 \begin{alltt}
-  <dependencies>
-    ... existing <dependency> items ...
+./gradlew deployArtifactsToLocalRepo
+\end{alltt}
 
-    <!-- Annotations from the Checker Framework: nullness, interning, locking, ... -->
-    <dependency>
-      <groupId>org.checkerframework</groupId>
-      <artifactId>checker-qual</artifactId>
-      <version>\ReleaseVersion{}</version>
-      <scope>system</scope>
-      <systemPath>\$\{env.CHECKERFRAMEWORK\}/checker/dist/checker-qual.jar</systemPath>
-    </dependency>
-    <dependency>
-      <groupId>org.checkerframework</groupId>
-      <artifactId>checker</artifactId>
-      <version>\ReleaseVersion{}</version>
-      <scope>system</scope>
-      <systemPath>\$\{env.CHECKERFRAMEWORK\}/checker/dist/checker.jar</systemPath>
-    </dependency>
-    <!-- The annotated JDK to use. -->
-    <dependency>
-      <groupId>org.checkerframework</groupId>
-      <artifactId>jdk8</artifactId>
-      <version>\ReleaseVersion{}</version>
-      <scope>system</scope>
-      <systemPath>\$\{env.CHECKERFRAMEWORK\}/checker/dist/jdk8.jar</systemPath>
-    </dependency>
-    <dependency>
-    <groupId>com.google.errorprone</groupId>
-      <artifactId>javac</artifactId>
-      <version>9+181-r4173-1</version>
-    </dependency>
-  </dependencies>
+\noindent
+Then, in the project you want to type-check, use
+\<\ReleaseVersion{}-SNAPSHOT> as the version number:
+
+\begin{alltt}
+   <artifactId>maven-compiler-plugin</artifactId>
+      ...
+      <configuration>
+        ...
+        <annotationProcessorPaths>
+          <path>
+            <groupId>org.checkerframework</groupId>
+            <artifactId>checker</artifactId>
+            <version>\ReleaseVersion{}-SNAPSHOT</version>
+          </path>
 \end{alltt}
 
+You can make similar changes if you have changed other artifacts besides
```

Comment:
It may be least error-prone just to say "use \ReleaseVersion{}-SNAPSHOT everywhere that the above instructions say \ReleaseVersion".  That is less ambiguous, less error-prone, and requires less thought from the reader.

Reply:
`\ReleaseVersion{}-SNAPSHOT` is the snapshot before the release, so it's not the correct version to use.  `./gradlew version` outputs the version of the artifacts that `./gradlew deployArtifactsToLocalRepo` uses.

Reply:
Good point about the version number.  Can the text restructuring be done, with a correct version number?

Reply:
(That is, is the text restructuring a good idea?)